### ISSUE TITLE
variable names are same, but still getting this error. Can anyone help

### ISSUE THREAD

**Issue by samantaraymadhu6** [walkerke/tidycensus#295]
pop90_10 <- map_dfc(c(1990,2000), function(x) {
get_decennial(geography ="state",
 variables = c(totalpop = "POP100"),
dataset = "sf1",
year = x) %>%
mutate(year = x)}
)
Getting data from the 1990 decennial Census
Error : Your API call has errors.  The API message returned is There was an error while running your query.  We've logged the error and we'll correct it ASAP.  Sorry for the inconvenience..
Error in gather.default(., key = variable, value = value, -GEOID, -NAME) : 
  object 'NAME' not found

**Comment by mfherman** [comment]
Hi @samantaraymadhu6! Last week, the Census turned off the API endpoints for 1990 and 2000 census data. They moved 2000 SF1 to a new endpoint, but the others remain inaccessible via API at this time. See #293 and #289 for more discussion.

We have updated the GitHub version of the package to use the new 2000 endpoint, so if you just need 2000 data, you could use the following code once you update:

``` r
library(tidycensus)

get_decennial(
  geography = "state",
  variables = c(totalpop = "P001001"),
  dataset = "sf1",
  year = 2000
  ) 
#> Getting data from the 2000 decennial Census
#> Using Census Summary File 1
#> # A tibble: 52 x 4
#>    GEOID NAME                 variable    value
#>    <chr> <chr>                <chr>       <dbl>
#>  1 25    Massachusetts        totalpop  6349097
#>  2 37    North Carolina       totalpop  8049313
#>  3 24    Maryland             totalpop  5296486
#>  4 12    Florida              totalpop 15982378
#>  5 11    District of Columbia totalpop   572059
#>  6 15    Hawaii               totalpop  1211537
#>  7 27    Minnesota            totalpop  4919479
#>  8 09    Connecticut          totalpop  3405565
#>  9 10    Delaware             totalpop   783600
#> 10 23    Maine                totalpop  1274923
#> # ... with 42 more rows
```

<sup>Created on 2020-09-15 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by samantaraymadhu6** [comment]
thanks. I tried the above code ..It still didn't work.

> get_decennial(
+   geography = "state",
+   variables = c(totalpop = "P001001"),
+   dataset = "sf1",
+   year = 2000
+ ) 
Getting data from the 2000 decennial Census
Error : Your API call has errors.  The API message returned is There was an error while running your query.  We've logged the error and we'll correct it ASAP.  Sorry for the inconvenience..
Error in gather.default(., key = variable, value = value, -GEOID, -NAME) : 
  object 'NAME' not found
